features/accommodations/creation/HouseRulesForm.tsx

```diff
@@ -1,5 +1,22 @@
 import { type FC } from "react";
+import { type CreateAccommodation } from "../createAccommodation";
 
-export const HouseRulesForm: FC = () => {
-  return <>House rules</>;
+type HouseRulesFormProps = {
+  updateFields: (fields: Partial<CreateAccommodation>) => void;
+  houseRules: string;
+};
+
+export const HouseRulesForm: FC<HouseRulesFormProps> = ({ updateFields, houseRules }) => {
+  return (
+    <>
+      <h3 className="text-xl mb-6 font-semibold">Set house rules for your guests</h3>
+      <textarea
```

Comment:
Do you have same textarea across you forms? If you have, you can extract it like you did for simple text input.